tests/test_app.py

```diff
@@ -0,0 +1,26 @@
+from importlib import reload
+
+import lambdas
+
+
+def test_app_configures_sentry_if_dsn_present(
+    caplog,
+    monkeypatch,
+):
+    monkeypatch.setenv("SENTRY_DSN", "https://1234567890@00000.ingest.sentry.io/123456")
+    reload(lambdas)
+    assert (
+        "Sentry DSN found, exceptions will be sent to Sentry with env=test" in caplog.text
+    )
+
+
+def test_app_doesnt_configure_sentry_if_dsn_not_present(caplog, monkeypatch):
+    monkeypatch.delenv("SENTRY_DSN", raising=False)
+    reload(lambdas)
+    assert "No Sentry DSN found, exceptions will not be sent to Sentry" in caplog.text
+
+
+def test_lambda_handler_success(lambda_function_event_payload):
+    reload(lambdas)
+    response = lambdas.lambda_handler(lambda_function_event_payload, {})
+    assert response["body"] == "Hello, World! WORKSPACE=test"
```

Comment:
This is somewhat of a general/specific comment.

**General**: what do you think about breaking up tests for the Flask webapp itself vs the AWS lambda handler function?

I could see tests like:
  * Flask App
    * testing `ping` style route for HTML response
    * test routes for authentication redirects
  * Lambda Handler function
    * test structure of the response (always JSON)
    * look for status codes and

I would envision the lambda handler function tests would be fairly minimal.  Just enough to establish the structure of the response, maybe some simple cases, but not as in-depth as the Flask webapp which is more concerned with the "correctness" of the application logic.

**Specific**: for this test, I see that `response` is a JSON payload from the `asgi-wsgi` wrapped function,

```json
{
  "statusCode": 200,
  "cookies": [],
  "headers": {
    "content-type": "text/html; charset=utf-8",
    "content-length": "28"
  },
  "isBase64Encoded": false,
  "body": "Hello, World! WORKSPACE=test"
}
```

It might be nice -- and maybe this was planned -- to test the structure of this `asgi-wsgi` wrapped response; looking for `statusCode`, `body`, etc.

Reply:
See updates to [test_lambdas.py](https://github.com/MITLibraries/alma-sapinvoices-ui/pull/1/commits/2236bad54c1f5c142b8aa7db897efc6d017afe11#diff-6c39b9c335047ba0de8c5e1205d0c6b153979aea41196fc60630a06d68c89399) and [test_app.py](https://github.com/MITLibraries/alma-sapinvoices-ui/pull/1/commits/d15cd2a18a626d11f5ee206c375f70d9ab55dbe9#diff-a67cb1853203a6f1956991a9d9881d231c4d43557f5baffd45cc672a87e41cc6).